src/pages/Swap/index.tsx

```diff
@@ -546,9 +547,10 @@ export function Swap({
   const switchChain = useSwitchChain()
   const switchingChain = useAppSelector((state) => state.wallets.switchingChain)
   const showOptInSmall = !useScreenSize().navSearchInputVisible
+  const isDark = useIsDarkMode()
 
   const swapElement = (
-    <SwapWrapper chainId={chainId} className={className} id="swap-page">
+    <SwapWrapper isDark={isDark} className={className} id="swap-page">
```

Comment:
we changed the swap box to have a subtler glow on dark mode

Reply:
lgtm